### DISCUSSION TITLE
Running it using Colab's GPU ?

### DISCUSSION THREAD

**Discussion by ABBEN4** [hlky/stable-diffusion#82] [Q&A]
Hello ! 

First, great job on this tool. I am very much about UI/UX and this is indispensible.

Secondly, as many I rely on colab to get enough GPU to run whatever I need from AI tools / run my models. Thus is there any way to run the whole on colab ?

Thank you,

**Comment by tudoanh** [comment]
Try this

https://github.com/altryne/sd-webui-colab

**Comment by listelin** [comment]
Similar to https://github.com/altryne/sd-webui-colab, but this also works.
https://colab.research.google.com/drive/1cd35l21ewU0fwWEnPjY_th5YORmMfZCd

The downside is that it takes a very long time to start up.
